src/index.ts

```diff
@@ -9,9 +9,9 @@ import { downloadBinary, findRelease } from "./release"
 async function main() {
   if (!(await isReady())) {
     try {
-      const [name, assets] = await findRelease(VERSION)
+      const [name, asset_id, asset_filetype] = await findRelease(VERSION)
```

Comment:
nitpick (not blocking at all), but just saying, we should use the convention of the language for naming variables and in JavaScript it's camelCase.